build/commands/lib/syncUtils.js

```diff
@@ -130,7 +130,7 @@ function syncChromium(syncWithForce, sync_chromium, delete_unused_deps) {
   const requiredChromiumRef = config.getProjectRef('chrome')
   let args = [
     'sync', '--nohooks', '--revision', 'src@' + requiredChromiumRef, '--reset',
-    '--with_tags', '--with_branch_heads', '--upstream'
```

Comment:
@cdesouza-chromium I was running all my working directories in this setup for a week now and haven't experienced any issues. I also did a clean Brave checkout with a an empty git cache, things were good.

Can you please try this change manually in your `canary` branch flow for a few days? If Chromium bumps won't trigger any `gclient sync` issues, I believe we can merge this safely.

Reply:
Will pull this into Canary. No worries.

Reply:
If I'm not mistaken that changes results to inability to sync some chromium revisions.
The issues was that some third_party HEADs were set to those branch_heads and `gclient sync` doesn't  any code to fetch them.
I hope the situation is changed now.

The instruction still referrers the switches for working with release branches:
https://www.chromium.org/developers/how-tos/get-the-code/working-with-release-branches/

Reply:
My old attempt to remove them: https://github.com/brave/brave-core/pull/11526#discussion_r787843987

Reply:
> If I'm not mistaken that changes results to inability to sync some chromium revisions. The issues was that some third_party HEADs were set to those branch_heads and `gclient sync` doesn't any code to fetch them. I hope the situation is changed now.

yep, this exact issue should be fixed now:
https://chromium.googlesource.com/chromium/tools/depot_tools.git/+/dbbe224787188ca748eef54e44d6651c1c90b90b

Reply:
> Will pull this into Canary. No worries.

@cdesouza-chromium I assume there were no issues?

Reply:
I have seen any issues so far, yeah.